src/Map.js

```diff
@@ -1,16 +1,31 @@
 import React, { Component } from 'react';
-import { withGoogleMap, GoogleMap } from 'react-google-maps';
+import { withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+import PropTypes from 'prop-types';
 
-const MyMap = withGoogleMap(() => (
+const MyMap = withGoogleMap(props => (
   <GoogleMap
-    defaultZoom={4}
-    defaultCenter={{ lat: 25.7392, lng: -104.9903 }}
-  />
+    defaultZoom={14}
+    defaultCenter={{ lat: 35.6857933, lng: 139.7501793 }}
+    onClick={props.onMapClick}
+  >
+    {props.markers.map((marker) => {
+      const position = {
+        lat: marker.lat,
+        lng: marker.lng,
+      };
+      return (<Marker
+        position={position}
+        key={position.lat + position.lng}
```

Comment:
`position.lat + position.lng` could be the same. can we use concat instead of add?

Reply:
Good catch